test/README.md

```diff
@@ -0,0 +1,103 @@
+# Running Tests
+
+To run all tests, perform the following steps:
+
+## Startup a fresh iRODS server
+
+The iRODS testing environment is assumed to be used to setup an iRODS server, with a minimum version of 4.3.2.
+See the testing environment to setup a clean iRODS server: https://github.com/irods/irods_testing_environment
+
+### Get the iRODS network information
+
+We need the network that the HTTP API is running on to have all pieces of the server
+communicate with each other.
```

Comment:
Consider rephrasing this statement so it's clear to the reader what is being talked about.
```
to have all pieces of the server￼communicate with each other.
```

What does `the server` refer to? The HTTP API server or the iRODS server?
What does `pieces` mean / refer to?

Reply:
Done.